support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutForm.test.jsx

```diff
@@ -74,6 +76,10 @@ describe('Digital checkout form', () => {
       },
     };
 
+    window.fetch = async () => ({
```

Comment:
Is this to mock the stripe setup intent?

Reply:
Yes that's right 👍